src/api/vaas.ts

```diff
@@ -0,0 +1,38 @@
+import {enabledChains} from "../chains.ts";
+import {mockWormhole} from "../mockGuardian.ts";
+import {type Request, type Response} from "express";
+import type {Hex} from "viem";
+
+export const vaasHandler = async (req: Request, res: Response) => {
+    let txHash = req.query["txHash"];
+
+    if (!txHash) {
+        res.status(400).send("txHash is required.");
+        return;
+    }
+
+    // Loop through enabledChains and try mockWormhole for each one, returning the first signed VAA
+    // that mockWormhole returns.
+    for (const chainConfig of Object.values(enabledChains)) {
+        try {
+            const result = await mockWormhole(
+                chainConfig.rpc,
+                txHash as Hex,
```

Comment:
typically I recommend avoiding `as` and would prefer to use `isHex` to ensure these values are hex. the `mockWormhole` method requires hex as a downstream of using viem and only handles EVM right now so I'm alright with saving this for a future fix.